src/index.html

```diff
@@ -9,10 +9,135 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
The class names 'calendar--start-day-sun' and 'calendar--month-length-31' are not flexible. If you want to change the start day or the number of days in a month, you would have to change the class names in the HTML and CSS. Consider using JavaScript to add these classes dynamically based on the current month.

Reply:
That is strange, cause the Dom module lessons are not covered yet. They are blocked